Sources/ArcGISToolkit/Components/FloatingPanel/FloatingPanel.swift

```diff
@@ -75,6 +74,7 @@ struct FloatingPanel<Content>: View where Content: View {
                         Divider()
                     }
                     content()
+                        .padding(.bottom, isCompact ? keyboardHeight - geometryProxy.safeAreaInsets.bottom : .zero)
```

Comment:
Padding the bottom of the content with an amount equal to the height of the keyboard gives us manual avoidance.

We need to take away the bottom safe area height because this height is included in the keyboard frame height on devices with rounded screen corners.